vignettes/standardize_parameters_effsize.Rmd

```diff
@@ -407,152 +417,5 @@ m2 <- lm(salary ~ xtra_hours + n_comps + seniority, data = hardlyworking)
 cohens_f_squared(m1, model2 = m2)
 ```
 
-<!-- ## Methods Comparison -->
-
```

Comment:
@mattansb I removed this section and put it into the vignette in the WIP folder, so it's not lost, but also not cluttering up this vignette.

Reply:
I actually think this is important, to show how these differ.
Cohen's *f* can be dropped here, though.

Reply:
Yup, looks good.

I would also maybe remove Cohen's *f* - seems like an effectsize / performence issue.